### PR TITLE
Fix webhook configuration of ssp

### PR DESCRIPTION
**Pull request by erkanerol**
Signed-off-by: Erkan Erol <eerol@redhat.com>

```release-note
NONE
```

### CODE REVIEW

File: deploy/webhooks.yaml [hunk 1]
```diff
@@ -124,7 +124,7 @@ webhooks:
       name: ssp-operator-service
       namespace: kubevirt-hyperconverged
       path: /validate-ssp-kubevirt-io-v1beta1-ssp
-      port: 8443
+      port: 9443
```
Comment (zcahana) [inline deploy/webhooks.yaml:127]: Good catch. And +1 for auto-generating this file sometime in the future :-)

File: deploy/webhooks.yaml [hunk 2]
```diff
@@ -139,5 +139,6 @@ webhooks:
     - UPDATE
     resources:
     - ssps
+    scope: Cluster
```
Comment (zcahana) [inline deploy/webhooks.yaml:142]: The SSP CRD isn't cluster-scoped, see here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/20df7b1db67c06d4d7edc3fe362d2be8e09e27fe/deploy/crds/scheduling-scale-performance00.crd.yaml#L15

File: deploy/operator.yaml [hunk 3]
```diff
@@ -650,7 +650,7 @@ spec:
     protocol: TCP
     targetPort: 9443
   selector:
-    name: ssp-operator
+    control-plane: ssp-operator
```
Comment (zcahana) [inline deploy/operator.yaml:653]: This file is autogenerated, so fixing it inline won't do - probably need to fix this at the manifest-templator.

**Review by zcahana — CHANGES_REQUESTED** [review]

File: deploy/operator.yaml (hunk 3 above)
Comment (erkanerol) [reply deploy/operator.yaml:653]: done

File: deploy/webhooks.yaml (hunk 2 above)
Comment (erkanerol) [reply deploy/webhooks.yaml]: reverted

**Comment by zcahana** [comment]
/lgtm

Since we don't have any non-OLM lane that exercises the SSP webhook (created in OpenShift only), I guess we can make do for now with manual verification that this webhook is now usable.
@tiraboschi should we be adding a non-OLM lane that uses `deploy.sh` on OpenShift?

**Pull request merged**